src/main/java/gomedic/model/AddressBook.java

```diff
@@ -19,6 +21,7 @@
 
     private final UniquePersonList persons;
```

Comment:
You might want to change this to UniquePersonList now and replaces all old `UniquePersonList`, I think it can be done now

Reply:
ok i will make a separate pr for tht